timestamp-task/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+		<artifactId>spring-cloud-task-modules</artifactId>
+		<groupId>org.springframework.cloud.task.module</groupId>
+		<version>${task-modules-version}</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.springframework.cloud.task.module</groupId>
+	<artifactId>timestamp-task</artifactId>
+	<packaging>jar</packaging>
+	<name>timestamp-task</name>
+	<description>Spring cloud timestamp task</description>
+
+	<properties>
+		<start-class>org.springframework.cloud.task.module.timestamp.TimestampTaskApplication</start-class>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-batch</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-jdbc</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.hsqldb</groupId>
+			<artifactId>hsqldb</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-config</artifactId>
+			<version>${cloud-config.version}</version>
+		</dependency>
```

Comment:
Does it make sense to move this dependency up a level?  What modules would _not_ need it?